tests/fixtures/controllers/getController.ts

```diff
@@ -40,6 +40,12 @@ export class GetTestController extends Controller {
     modelsArray: new Array<TestSubModel>(),
     numberArray: [1, 2, 3],
     numberValue: 1,
+    object: {
```

Comment:
are these necessary for the tests to compile?

Reply:
Yea, unfortunately they are necessary for the tests to compile. It happened because one of the previous PRs got merged without the tests running (woops!). But now that the previous PR added husky, these kind of things will be caught.